src/lib/ui/Switch.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react'
+
+interface SwitchProps {
+  defaultChecked?: boolean
+  label?: boolean
+  onChange?: (checked: boolean) => void
+}
```

Comment:
What if we make this a controlled component and let the consumer manage the state? They will need the current value anyway, so better to let them pass it in.

```suggestion
interface SwitchProps {
  checked: boolean
  onClick?: () => void
  label?: boolean
}
```

Reply:
This suggestion might be off base if our general approach was to [mimic the MUI api though](https://mui.com/material-ui/react-checkbox/). CC @mikewuu

Reply:
This is all internal though so non-blocking and simple if we change our mind. Just good to align on this as we build out more ui lib component.

Reply:
Controlled only makes sense IMO since this isn't a general UI lib. I would go a step further, and make `onChange` required, and accept the new state too:

```
onChange: (checked: boolean) => void
```

Reply:
[62ade83](https://github.com/seamapi/react/pull/377/commits/62ade83739729d20cd582c4dadf95ef35952c179)

Reply:
wait i want this toggle to work in the story one sec

Reply:
[228aa0d](https://github.com/seamapi/react/pull/377/commits/228aa0df22c05f389c9d8015ea77221e9aa6bb70)